wolfcrypt/src/asn.c

```diff
@@ -25321,8 +25321,8 @@ int wc_DerToPemEx(const byte* der, word32 derSz, byte* output, word32 outSz,
     char header[MAX_X509_HEADER_SZ + HEADER_ENCRYPTED_KEY_SIZE];
     char footer[MAX_X509_HEADER_SZ];
 #endif
-    int headerLen = MAX_X509_HEADER_SZ + HEADER_ENCRYPTED_KEY_SIZE;
-    int footerLen = MAX_X509_HEADER_SZ;
+    size_t headerLen = MAX_X509_HEADER_SZ + HEADER_ENCRYPTED_KEY_SIZE;
```

Comment:
You are now casting in lots more places than before.
Is there a need for it to be size_t?

Reply:
I was actually able to _remove_ a bunch of casts by making those `size_t`s, which is what they natively are -- they're assigned from `strlen()`.  The only truly right answer here is a more extensive rework to eliminate all the casts except for the final `return`ed values, and add range checking on the few remaining casts.  That is a bear of a task for another day!